isis/src/qisis/objs/CubeViewport/CubeViewport.h

```diff
@@ -339,7 +340,7 @@ namespace Isis {
        */
       double grayPixel(int sample, int line);
       //! @return The gray Stretch
-      Stretch grayStretch() const;
+      CubeStretch grayStretch() const;
       //! @return The red Stretch
       Stretch redStretch() const;
```

Comment:
Again, why is grayStretch a CubeStretch and red/green/blueStretch not? Is it just that this is not intended to save stretches of RGB images?

Reply:
This was an oversight because originally this was just designed to work with gray stretches. Thanks for catching this and pointing it out!